### DISCUSSION TITLE
[Projects Beta] Labels are missing from board view

### DISCUSSION THREAD

**Discussion by amacneil** [community/community#5843] [Projects and Issues] [answered]
In the old Github Issues project board, labels for each issue/PR were shown. In the new issues beta, I can't see the labels nor any way to enable this feature.

Am I missing a setting somewhere or is this just a regression in functionality?

**Comment by patrickklaeren** [comment, marked as answer]
You need to enable them on the board/list per the settings.

**Reply by amacneil** [reply]
Thank you! I was looking in the project settings, but I didn't notice this under the specific board settings.

**Reply by cuddihyge** [reply]
Where?  in which settings?   There is not enough detail to help me.

**Reply by patrickklaeren** [reply]
I highly advise you just click around and explore Projects. There's not a lot to go through and it's not like you can break anything.

As for this specific question, it's just another column:
![image](https://user-images.githubusercontent.com/1341180/196007760-0ab2967d-ddc7-45d8-8a2e-25136b5a9dc5.png)

**Comment by mikkokotila** [comment]
I think this answer is out-dated. Currently, if you are in the board, there is no way to do this, so you have to go to table, and then do it there, and come back.

**Reply by emanoelqueiroz** [reply]
Has a little button in the side of the View where you can configure all those things

![image](https://github.com/community/community/assets/6198732/4e6de6a6-45aa-49ae-92de-68c650c36e98)

**Reply by JasonToups** [reply]
@mikkokotila , when you are in your **Project**, there are several _Tabs_ across, which are your different _Views_ of the issues.
If you **select a View**, there's a _dropown menu_ on that Tab.
Select the **Dropdown Menu**, and you will see the _list of fields_ you can show on the cards.
![portfolio-project-view](https://github.com/community/community/assets/8608152/1db84d66-9e8c-44cc-b6f1-e842ec8ae241)

Then you will see the **Notification Badge** on the _Dropdown Menu_.
![portfolio-project-save](https://github.com/community/community/assets/8608152/061a5b55-0289-483f-9d60-3258e466acb7)
So you have to look to the _right of the View_ for the **Save button**.
Click the Save button, and that will _save your Project View Configuration_.

Cheers,
And [Checkout my Portfolio](https://jasontoups.github.io/) to see what I've been working on.
